src/redux/actions/genes/loadDownsampledGeneExpression.js

```diff
@@ -0,0 +1,157 @@
+import _ from 'lodash';
+import { SparseMatrix } from 'mathjs';
+
+import {
+  GENES_EXPRESSION_ERROR,
+  DOWNSAMPLED_GENES_LOADING,
+  DOWNSAMPLED_GENES_LOADED,
+} from 'redux/actionTypes/genes';
+
+import fetchWork from 'utils/work/fetchWork';
+import getTimeoutForWorkerTask from 'utils/getTimeoutForWorkerTask';
+import { findLoadedGenes } from 'utils/genes';
+// import { findLoadedGenes } from 'utils/genes';
+
+// Debounce so that we only fetch once the settings are done being set up
+const loadDownsampledGeneExpressionDebounced = _.debounce(
```

Comment:
Debounced so that we don't trigger a lot of expression load requests in close succession, rather wait until the last one. (it's easy for many to happen while reordering group by's)